src/blueprints/tv2/timeline-object-factories/tv2-atem-video-mixer-timeline-object-factory.ts

```diff
@@ -289,25 +294,26 @@ export class Tv2AtemVideoMixerTimelineObjectFactory implements Tv2VideoMixerTime
     return atemMeTimelineObject.content.me.input
   }
 
-  public createCutTransitionEffectTimelineObjects(sourceInput: number): AtemMeTimelineObject[] {
+  public createCutTransitionEffectTimelineObjects(sourceInput: number, metadata?: Tv2TimelineObjectMetadata): AtemMeTimelineObject[] {
     const meContent: AtemMeTimelineObject['content']['me'] = {
       input: sourceInput,
       transition: AtemTransition.CUT
     }
     return [
-      this.createTransitionEffectTimelineObject(Tv2AtemLayer.PROGRAM, meContent),
-      this.createTransitionEffectTimelineObject(Tv2AtemLayer.CLEAN_FEED, meContent)
+      this.createTransitionEffectTimelineObject(Tv2AtemLayer.PROGRAM, meContent, metadata),
+      this.createTransitionEffectTimelineObject(Tv2AtemLayer.CLEAN_FEED, meContent, metadata)
     ]
   }
 
-  private createTransitionEffectTimelineObject(layer: Tv2AtemLayer, meContent: AtemMeTimelineObject['content']['me']): AtemMeTimelineObject {
+  private createTransitionEffectTimelineObject(layer: Tv2AtemLayer, meContent: AtemMeTimelineObject['content']['me'], metadata?: Tv2TimelineObjectMetadata): AtemMeTimelineObject {
     return {
       id: `${layer}_${meContent.transition}`,
       enable: {
         start: 0
       },
       layer,
       priority: 10,
+      metaData: metadata,
```

Comment:
```suggestion
      metadata,
```
Is it metaData or metadata? 😄 If we align the name, we won't need to assign it here like this, but could just do as the suggested here

Reply:
`metaData` is unfortunately how it is spelled in TSR. So in order for TSR to find the correct attribute, the attribute has to be named `metaData` on the `TimelineObject`.